components/ProfileStarComments.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Star as StarType } from '../graphql/index'
+import { Star as StarIcon } from 'react-feather'
+import _ from 'lodash'
+import styles from '../scss/profileStarComments.module.scss'
+
+type StarCommentsProps = {
+  stars: StarType[]
+}
+
+type StarCommentProps = {
+  star: StarType
+}
+
+const StarComment: React.FC<StarCommentProps> = ({ star }) => {
+  const fullname = _.get(star, 'student.name', '')
+  const username = _.get(star, 'student.username', '')
+  const lessonTitle = _.get(star, 'lesson.title', '')
+  const starComment = _.get(star, 'comment', '')
+  const firstName = fullname.split(' ')[0] || 'A'
+  const lastName = fullname.split(' ')[1] || ' '
+  return (
+    <div className={`${styles['comment-box']} shadow-sm`}>
+      <div className={`${styles['comment-profile-container']}`}>
+        <div className={`${styles['user-info-wrapper']}`}>
+          <div
+            className={`${styles['user-info-image']} text-uppercase bg-primary rounded-circle text-light`}
+          >
+            {firstName[0]}
+            {lastName[0]}
+          </div>
+          <div>
+            <div className={`${styles['comment-username']} text-left`}>
+              {'@' + username}
+            </div>
+            <h6 className={`${styles['comment-lesson-title']} text-left`}>
```

Comment:
It's a very minor thing since c0d3 is not really optimized for accessibility but skipping heading ranks should be avoided if possible. [See](https://www.w3.org/WAI/tutorials/page-structure/headings/)

Reply:
I changed everything to h6 or p